modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -6,6 +6,16 @@
 	}
 }
 
+.form-control[type='text'] {
+	background-color: #e7e7ed;
+}
+.has-error .form-control[readonly] {
+	background-color: #e7e7ed;
+}
+.has-success .form-control[readonly] {
+	background-color: #e7e7ed;
+}
+
```

Comment:
Hi @Valmir-unicap,

please, could you make this adjustments? This keep the red colors related to the field errors. Thanks!

```suggestion
.form-control[readonly] {
	background-color: #f1f2f5;
        border-color: #e7e7ed;
	cursor: not-allowed;
}

.has-success .form-control[type='text'] {
	background-color: #f1f2f5;
	border-color: #e7e7ed;
}

.has-error .form-control[readonly] {
	background-color: #feefef;
        border-color: #f48989;
}

```

Reply:
Corrected

Reply:
Thank you :)